### ISSUE TITLE
Resumable is not defined

### ISSUE THREAD

**Issue by fitnessmechanics** [dilab/resumable.php#24]
I am trying to learn to work with Resumable.js, and I keep coming up with the following error:

ReferenceError: Resumable is not defined.

In addition, clicking the h1 browseButton tag does not give a response. I have not made any changes to the resumable.js file, and my html in the test file is as follows:

```
<script src="resumable.js" type="application/javascript"></script>
<script>
var r = new Resumable({
  target: 'phptest.php',
});


r.assignBrowse(document.getElementById('browseButton'));
r.assignDrop(document.getElementById('browseButton'));

r.on('fileSuccess', function(file){
    console.debug('fileSuccess',file);
  });
r.on('fileProgress', function(file){
    console.debug('fileProgress', file);
  });
r.on('fileAdded', function(file, event){
    r.upload();
    console.debug('fileAdded', event);
  });
r.on('filesAdded', function(array){
    r.upload();
    console.debug('filesAdded', array);
  });
r.on('fileRetry', function(file){
    console.debug('fileRetry', file);
  });
r.on('fileError', function(file, message){
    console.debug('fileError', file, message);
  });
r.on('uploadStart', function(){
    console.debug('uploadStart');
  });
r.on('complete', function(){
    console.debug('complete');
  });
r.on('progress', function(){
    console.debug('progress');
  });
r.on('error', function(message, file){
    console.debug('error', message, file);
  });
r.on('pause', function(){
    console.debug('pause');
  });
r.on('cancel', function(){
    console.debug('cancel');
  });
</script>
<h1 id="browseButton">Select files</h1>
```

**Comment by dilab** [comment]
Is file resumable.js stored locally correctly ?

**Comment by fitnessmechanics** [comment]
It's in the same folder as the file calling it, is there more to it that I'm missing?

**Comment by dilab** [comment]
Put the JS script at the bottom of the page. Make it the last piece to run.

**Comment by fitnessmechanics** [comment]
Tried this, didn't change the result.

**Comment by fitnessmechanics** [comment]
I took a whack at this today in at least 3 different browsers all running the console inspector. Eventually found at line 378, written in my code editor like this: 

(extension.indexOf('*') !== -1 && fileType.substr(0, extension.indexOf('*')) === extension.substr(0, extension.indexOf('*')))  ||

turned into this:

(extension.indexOf('*') !== -1 && fileType.substr(0, extension.indexOf('*')) === extension.substr(0, extension.indexOf('*'))) Â ||

Notice the extra accented A near the end?
I found that removing the spaces on either side of where the A appears seemed to prevent the A from appearing and solve the error, although I have yet to fully test the API, I am hopeful that the issue has been resolved.

Also, I realize now that this is a forum for the server side PHP code, I apologize that it is in the wrong place.
